src/screens/Portfolio/index.js

```diff
@@ -86,36 +100,47 @@ class Portfolio extends Component<
     // TODO pull to refresh connected to bridge (need to think it modular so we can reuse easily)
 
     return (
-      <View style={styles.root}>
-        <StatusBar barStyle="dark-content" />
-
-        <SectionList
-          sections={(sections: any)}
-          style={styles.sectionList}
-          ListHeaderComponent={GraphCardContainer}
-          ListFooterComponent={
-            !completed
-              ? LoadingFooter
-              : sections.length === 0
-                ? NoOperationFooter
-                : NoMoreOperationFooter
-          }
-          refreshControl={<SyncRefreshControl />}
-          keyExtractor={this.keyExtractor}
-          renderItem={this.renderItem}
-          renderSectionHeader={SectionHeader}
-          onEndReached={this.onEndReached}
-          showsVerticalScrollIndicator={false}
-        />
-      </View>
+      <ComputeSummary>
+        {summary => (
+          <View style={styles.root}>
+            <StatusBar barStyle="dark-content" />
+            <AnimatedSectionList
+              sections={sections}
+              style={styles.sectionList}
+              keyExtractor={this.keyExtractor}
+              renderItem={this.renderItem}
+              renderSectionHeader={SectionHeader}
+              onEndReached={this.onEndReached}
+              showsVerticalScrollIndicator
+              scrollEventThrottle={16}
+              onScroll={Animated.event(
+                [{ nativeEvent: { contentOffset: { y: scrollY } } }],
+                { useNativeDriver: true },
+              )}
+              ListHeaderComponent={() => (
```

Comment:
this has to be a "static" component, if not, we're going to have unmount/remount everytime because how React reconcilier works.

I would suggest that the ComputeSummary is not dynamically done in a closure, but instead we do a HOC that send the data by props.

therefore you can have a class field `ListHeaderComponent = () => <GraphCardContainer summary={this.props.summary} />`